pulp_smash/cli.py

```diff
@@ -326,25 +335,34 @@ def start(self, services):
         :return: An iterable of :class:`pulp_smash.cli.CompletedProcess`
             objects.
         """
-        if self._svc_mgr == 'sysv':
-            with self._disable_selinux():
-                return self._start_sysv(services)
-        elif self._svc_mgr == 'systemd':
-            return self._start_systemd(services)
-        else:
-            raise NotImplementedError(
-                'Service manager not supported: {}'.format(self._svc_mgr)
-            )
-
-    def _start_sysv(self, services):
+        services = set(services)
+        for system in self._cfg.systems:
+            intersection = services.intersection(
+                self._cfg.services_for_roles(system.roles))
+            if intersection:
+                client = Client(self._cfg, pulp_system=system)
+                svc_mgr = self._get_service_manager(self._cfg, system)
+                if svc_mgr == 'sysv':
+                    with self._disable_selinux(client):
+                        return self._start_sysv(client, intersection)
```

Comment:
> I think the former may be more interesting since we can control which pulp_system we want to stop a service and then properly test the fail over features.

Great point. We need the ability to manage services on a single system.